packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -1645,8 +1643,6 @@ describe('network stubbing', function () {
       }).then(function () {
         $.get('/post-only').done((responseText) => {
           expect(responseText).to.contain(body)
-
-          done()
```

Comment:
Fixes  https://github.com/cypress-io/cypress/issues/23264. We didn't need `done()` here -- passing and failing test cases work as expected here without calling `done()`